content/en/docs/reference/kubernetes-api/common-parameters/common-parameters.md

```diff
@@ -58,6 +58,30 @@ When present, indicates that modifications should not be persisted. An invalid o
 
 
 
```

Comment:
huh... the header says this is auto-generated:
`The file is auto-generated from the Go source code of the component using a generic...`

not sure if that's true or where the source for this is

Reply:
current content lgtm though

Reply:
hmm, that's a good question. The other parameter descriptions do look like they are pulled from the comments of `meta/v1/types.go`. 

I tried following the instructions for [updating the generated docs](https://github.com/kubernetes/website/blob/main/content/en/docs/contribute/generate-ref-docs/quickstart.md) and was successfully able to run `./update-imported-docs.py`, but all it did was update `kubectl-commands.html` with an unrelated change. I think one issue may be that I can't use a branch that [doesn't exist yet ](https://github.com/kubernetes/website/blob/main/update-imported-docs/reference.yml#L10)

@mehabhalodiya can you confirm that these docs are still being auto-generated and that the source of truth is `meta/v1/types.go`? If so, do you know how to best update these so that my [new changes](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L557-L568) are included here?

Reply:
@kevindelgado I too believe, `The file is auto-generated` as mentioned by @liggitt @sftim .
Also, may be you can set the branch of the previous release until 1.23 isn't formally released.